package.json

```diff
@@ -51,11 +51,13 @@
     "@types/react-dom": "npm:types-react-dom@beta",
     "@typescript/lib-dom": "npm:@types/web@^0.0.183",
     "buffer": "^6.0.3",
+    "bundler-upload-sdk": "^2.1.1",
```

Comment:
please do not install those dependencies at the root project.

Reply:
got you...moved them!